test/e2e/query_tickets_test.go

```diff
@@ -155,7 +282,7 @@ func TestQueryTickets(t *testing.T) {
 				pageCounts := 0
 				ctx := om.Context()
 
-				test.preAction(ctx, fe, t)
+				doCreateTicket(ctx, fe, test.gotTickets, t)
```

Comment:
Why are you declaring doCreateTickets and using it once instead of just having the code it contains here?